jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -219,38 +210,38 @@ public synchronized ResponseEntity<String> depositTim(String jsonString, Request
     // Setting the SerialId to OdeBradcastTim serialId to be changed to
```

Comment:
Issue (non-blocking): Typo here, "OdeBradcastTim" should probably be "OdeBroadcastTim"